python_modules/dagster-graphql/dagster_graphql/implementation/events.py

```diff
@@ -309,6 +310,8 @@ def from_dagster_event_record(event_record: EventLogEntry, pipeline_name: str) -
         return GrapheneObservationEvent(
             event=event_record,
         )
+    elif dagster_event.event_type == DagsterEventType.ASSET_FAILED_TO_MATERIALIZE:
```

Comment:
Can we look into adding a lint rule or a test or something similar that makes sure new DagsterEventType enum values are appropriately covered here?